reco_utils/recommender/newsrec/models/base_model.py

```diff
@@ -36,13 +38,17 @@ def __init__(self, hparams, iterator_creator, seed=None):
             seed (int): Random seed.
         """
         self.seed = seed
-        tf.set_random_seed(seed)
+        tf.compat.v1.set_random_seed(seed)
```

Comment:
Same here. See above comment.

Reply:
this is to avoid the warnings, also in the future, if we decide to use tf2, this will work out of the box